lib/runner/reporters/junit.js

```diff
@@ -56,6 +56,22 @@ module.exports = new (function() {
 
     adaptAssertions(module);
 
+    var filename = moduleName + '.xml';
```

Comment:
I'm not really in favour of this change, I don't think it's necessary and the potential impact is unknown. I'd rather just stick to the classname attribute change without anymore changes.